cypress/e2e/marc/marc-bibliographic/edit-marc-bib/manual-linking/protocol-displayed-in-$0-of-manual-linked-field-with-base-url-without-protocol-before-linking.cy.js

```diff
@@ -25,7 +25,7 @@ describe('MARC', () => {
             'C436828 Tate Britain (Gallery)',
             'C436828 Action and adventure fiction',
           ],
-          url: 'http://id.loc.gov/',
+          url: 'https://',
```

Comment:
is this url right?

Reply:
Yeah, just need to check, if field has url in general, not specific url.